### ISSUE TITLE
Publish a container image for kcctl to DockerGub

### ISSUE THREAD

**Issue by gunnarmorling** [kcctl/kcctl#33]
Hey @aalmiray, is this something JReleaser would help with as well?

**Comment by aalmiray** [comment]
Yup, JReleaser can make it happen

**Comment by aalmiray** [comment]
@gunnarmorling we need the following information:

- a username/password for publishing to DockerHub. Store the password as a GitHub secret, I suggest using `KCCTL_DOCKER_PASSWORD`.
- a DockerHub repository. The computed default would be `kcctl/kcctl` but feel free to pick a different name.
- pick the base image. I think executables generated with Graal as we have them today require glibc, in which case that rules out any Linux Alpine base image. We could go with `scratch` or an Ubuntu based image.

**Comment by aalmiray** [comment]
Slight complication, the docker packager expects `.zip` or `.jar` files as inputs. We currently have `.tar.gz` for the Linux artifact. The quick workaround is to use the `.zip` produced by the build, making sure the zip file is not signed/released (which JReleaser 0.6.0 can do).

**Comment by gunnarmorling** [comment]
I'll try and register the kcctl/kcctl repo. Re the base image, the Quarkus build can produce an image itself (either based on RHEL UBI, or distroless).

**Comment by aalmiray** [comment]
That's the thing, at the moment JReleaser creates a Dockerfile based on a template, then creates the image, then published to a registry of your choice. It does not accept publishing a container image that was built from the outside, yet.

**Comment by gunnarmorling** [comment]
Ok, I see. I suppose we still can use the existing Dockerfile, right? One other question: can we deploy to quay.io instead of DockerHub? It seems the latter requires a paid plan now for setting up new organizations :(

**Comment by aalmiray** [comment]
Yes, we can deploy to Quai as well. You could setup a personal account at DockerHub, that's what I did for JReleaser, I did not migrate that account to org.

**Comment by aalmiray** [comment]
https://github.com/jreleaser/jreleaser/issues/358 Prevents the use of the `quay.io/quarkus/quarkus-distroless-image:1.0` image however `registry.access.redhat.com/ubi8/ubi-minimal:8.4` may be used. These two images are the ones defined by the Quarkus templates.

**Comment by mdwajid095** [comment]
Hello All,

I'm using kcctl as container. Really a nice, well readable tool for kafka connect.
Github: https://github.com/mdwajid095/wowsome/tree/main/kcctl
DockerHub: https://hub.docker.com/r/mdwajid095/kcctl

I think, we should include the docker image part in this project here itself. So, that everyone can use it. Thanks!
